src/Symfony/Component/RateLimiter/Tests/Policy/SlidingWindowTest.php

```diff
@@ -102,4 +102,13 @@ public function testGetRetryAfterUsesMicrotime()
         // should be 500ms left (10 - 9.5)
         $this->assertEqualsWithDelta(0.5, $window->getRetryAfter()->format('U.u') - microtime(true), 0.2);
     }
+
+    public function testCreateAtExactTime()
+    {
+        ClockMock::register(SlidingWindow::class);
+        ClockMock::withClockMock(1234567890.000000);
+        $window = new SlidingWindow('foo', 10);
+        $window->getRetryAfter();
+        $this->assertTrue(true);
```

Comment:
Any better assertion can be done? Maybe checking the value returned by `getRetryAfter`.

Reply:
I've changed it to better check - see the updated commit